apps/person-api/src/main/java/no/nav/registre/testnorge/person/provider/PersonController.java

```diff
@@ -28,10 +29,11 @@
 
     @PostMapping
     public ResponseEntity<Object> createPerson(
-            @RequestBody PersonDTO personDTO
+            @RequestBody PersonDTO personDTO,
+            @RequestParam(required = false, defaultValue = "DOLLY") String kilde
```

Comment:
Er dette optional i PDL kunne vi også hatt dette som optional? Er redd for at andre systemer kan begynne å logge som dolly